src/asmcnc/skavaUI/screen_go.py

```diff
@@ -463,70 +463,95 @@ def on_pre_enter(self, *args):
-            # Check brush use and lifetime: 
-            if self.m.spindle_brush_use_seconds >= 0.9 * self.m.spindle_brush_lifetime_seconds:
-                brush_use_string = "[b]" + str(int(self.m.spindle_brush_use_seconds / 3600)) + "[/b]"
-                brush_lifetime_string = "[b]" + str(int(self.m.spindle_brush_lifetime_seconds / 3600)) + "[/b]"
-
-                brush_warning = (
-                        self.l.get_bold("Check your spindle brushes before starting your job!") + "\n\n" + \
-                        (
-                            self.l.get_str(
-                                "You have used your SmartBench for N00 hours since you updated your spindle brush settings."
-                            ).replace(self.l.get_str('hours'), self.l.get_bold("hours"))
-                        ).replace("N00", brush_use_string) + \
-                        " " + \
-                        (
-                            self.l.get_str(
-                                "You have told us they only have a lifetime of N00 hours!"
-                            ).replace(self.l.get_str('hours'), self.l.get_bold("hours"))
-                        ).replace("N00", brush_lifetime_string)
-                )
-
-                brush_reminder_popup = popup_info.PopupReminder(self.sm, self.am, self.m, self.l, brush_warning,
-                                                                'brushes')
-
-            if self.m.time_since_z_head_lubricated_seconds >= self.m.time_to_remind_user_to_lube_z_seconds:
-                time_since_lubricated_string = "[b]" + str(
-                    int(self.m.time_since_z_head_lubricated_seconds / 3600)) + "[/b]"
-
-                lubrication_warning = (
-                        self.l.get_bold("Lubricate the z head before starting your job!") + "\n\n" + \
-                        (
-                            self.l.get_str(
-                                "You have used SmartBench for N00 hours since you last told us that you lubricated the Z head."
-                            ).replace(self.l.get_str('hours'), self.l.get_bold("hours"))
-                        ).replace("N00", time_since_lubricated_string) + \
-                        "\n\n" + \
-                        self.l.get_str("Will you lubricate the Z head now?") + "\n\n" + \
-                        self.l.get_str("Saying 'OK' will reset this reminder.")
-                )
-
-                lubrication_reminder_popup = popup_info.PopupReminder(self.sm, self.am, self.m, self.l,
-                                                                      lubrication_warning, 'lubrication')
+            # Statement below is evaluated left to right, so fw version has to be >= 2.2.8 for setting_51 to be checked
+            if self.m.is_machines_fw_version_equal_to_or_greater_than_version('2.2.8', 'Spindle lifetime check') and self.m.theateam() and self.m.s.setting_51:
```

Comment:
swap self.m.s.setting_51 for new [get_dollar_setting() function](https://github.com/YetiTool/easycut-smartbench/blob/415e3aaa27eef0745dd4450f192af62464fd989b/src/asmcnc/comms/router_machine.py#L1396), just protects in case setting_51 doesn't exist